### PR TITLE
Added API-contracts for/users/self route.

### PR DESCRIPTION
**Pull request by Rucha1499**
Added API-contracts for/users/self route.

### CODE REVIEW

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: users/README.md [hunk 1]
```diff
@@ -61,6 +62,31 @@ Returns all users in the system.
...
     - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
+## **GET /users/self**
+
+Returns the details of user.
+
+- **Params**  
+  None
+- **Query**  
+  size=[integer], page=[integer]
```
[the remark this answers is not in the document]
Comment (Rucha1499) [reply users/README.md]: Changes done!

File: users/README.md [hunk 2]
```diff
@@ -61,6 +62,29 @@ Returns all users in the system.
   - **Code:** 401
     - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
 
+## **GET /users/self**
+
+Returns the details of user.
```
Comment (swarajpure) [inline users/README.md:67]: Which user?
`Returns the details of the logged in user` makes more sense to me.

**Review by swarajpure — CHANGES_REQUESTED** [review]

File: users/README.md (hunk 2 above)
Comment (Rucha1499) [reply users/README.md:67]: Sure

**Review by whyDontI — APPROVED** [review]
Nice Work 👍🏽

**Pull request merged**